### PR TITLE
ES6: migrate "localization"

### PR DESCRIPTION
**Pull request by lipis**
I renamed the `strings.coffee` to `webapp.js` to have more unique names on [Crowdin](https://crowdin.com/project/wire-cs) (mainly because I don't want to lose all the translations there already. I already talked to the customer support of Crowdin and they were very helpful as recently there were some changes on their side and they don't support out of the box `*.js` files :) Long story short.. it works!

### CODE REVIEW

File: app/script/localization/Localizer.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  @param replace [Object | Array<Object>] Placeholders that should be replaced
+  @option replace [String] placeholder Content to be replaced
+  @option replace [String] content replacing content
+    */
+
+  get_text(valueAccessor) {
+    if (valueAccessor == null) return;
+
+    let args = [];
+    let s = valueAccessor;
```
Comment (lipis) [inline app/script/localization/Localizer.js:52]: better name for `s`.. any suggestions?

Comment (bennycode) [inline app/script/localization/Localizer.js:52]: There is no `@param id` nor a `@param replace`. I only see `@param valueAccessor`.

Comment (lipis) [reply app/script/localization/Localizer.js:52]: Next time comment under the right line so it will go away when fixed..

Comment (bennycode) [reply app/script/localization/Localizer.js:52]: Sure! I just didn't had my glasses on. :P

Comment (lipis) [reply app/script/localization/Localizer.js:52]: 👓

File: app/script/localization/Localizer.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      z.util.StorageUtil.set_value(z.storage.StorageKey.LOCALIZATION.LOCALE, param);
+    }
+    this.locale = z.util.StorageUtil.get_value(z.storage.StorageKey.LOCALIZATION.LOCALE) || navigator.language.substr(0, 2) || 'en';
+    moment.locale([this.locale, 'en']);
+    if (z.string[this.locale]) {
+      $.extend(z.string, z.string[this.locale]);
+    }
+  }
+
+  /*
```
Comment (gregor) [inline app/script/localization/Localizer.js]: JSDOC style /**

Comment (lipis) [reply app/script/localization/Localizer.js]: heheh :)

**Pull request merged**